src/Bounds.cpp

```diff
@@ -738,7 +738,8 @@ class Bounds : public IRVisitor {
                 // If the argument is unbounded on one side, then the max is unbounded.
                 max = Expr();
             }
-        } else if (op->is_intrinsic(Call::likely)) {
+        } else if (op->is_intrinsic(Call::likely) ||
+                   op->is_intrinsic(Call::likely_if_innermost)) {
```

Comment:
This way of doing it forces the likely flag to be constant. One can insist that the Expr in a second argument to the call is an IntImm, but that's getting into idiosyncratic constraints on IR.

I've thought of adding a new Annotate IR node for this sort of thing which would have one or more selectors and one Expr arg. It would always evaluate to the Expr but could be used for things like likely and maybe cache_tag. (The latter needs more than one Expr though.)

All in all I think two intrinsics here is best. Maybe we could add a "handle_pass_through_intrinsics" function to allow many visitors to handle the intrinsics which are defined to take one argument and return that argument as their result with one piece of code.